openstack/networking/v2/subnets/requests.go

```diff
@@ -108,6 +108,7 @@ type CreateOpts struct {
 	TenantID        string
 	AllocationPools []AllocationPool
 	GatewayIP       string
+	NoGateway       bool
```

Comment:
The zero-value for `bool` is `false`, so there's no way to check if a user set the field to `false` or just didn't provide it. With a `*bool`, we know that if the value is `true` or `false` it was provided by the user, because otherwise it'd be `nil`.